### ISSUE TITLE
Need a Mac to debug iOS Apps?

### ISSUE THREAD

**Issue by Chicagoan2016** [MarimerLLC/cslaforum#906]
**Question**
Hi All, 
I was asked to look into iOS apps and I started with an example but it seems to me  that we need a Mac, I do have an iPhone but I couldn't find a way to use it for debugging.
Do we have to have a Mac? if so, do we have to buy the latest version of Mac?
I know a Mac user will find my questions funny but I haven't used a Mac since 2003 (sweet memories from University of Texas :) )

Kind Regards

**Version and Platform**
CSLA version: 4.7.100
OS: Windows, Linux, iOS, Android, etc.
Platform: WinForms, WPF, ASP.NET Core, MVC, Xamarin, etc.

**Comment by rockfordlhotka** [comment]
Yes, you pretty much need to have a Mac. There's a service called MacInCloud (??) that you can use to access a Mac in the cloud, that might get you further.

@Bowman74 knows quite a lot about this sort of thing.

I have an old Macbook Air that I use. It is slow, but it works.
